docs/CxGO--(from-1.6.12).md

```diff
@@ -49,8 +51,46 @@ scan-preset | CSV of scanning rules
 ```
 java -jar <cx-flow-x.x.x.jar> --spring.config.location=application.yml --scan --cx-team="\my\bu\ba" --cx-project="Myprj" --app=AppID
 ```
-CLI execution mode is supported, and 
+CLI execution mode is supported
   * --scan indicates that the source will be zipped/scanned.  
   * --cx-team value must be the path where the project will be created - inclusive of the business application.  
```

Comment:
Better wording would be:
--cx-team value must be the path where the project will be created. The path includes business unit and application.